bump-version.sh

```diff
@@ -4,7 +4,7 @@
 # Version will be bumped in Tensile/__init__.py and in .yaml files
 
 OLD_VERSION="4.31.0"
```

Comment:
Does this need to be updated to "4.32.0" @zaliu?

Reply:
If we actually used the script, yes, we would change 4.31.0 to 4.32.0.  Otherwise the next feature complete merge will move 4.32.1 to OLD and 4.31.0 would be gone.  That said, it's a good point.